### PR TITLE
Adding SCP-2818

### PR DESCRIPTION
**Pull request by galaxy119**

### CODE REVIEW

**Comment by iopietro** [comment]
lore-wise SCP-2818 should shoot you and then kills you in the impact.

If we want to respect the lore, we could literally shoot the player who pressed the trigger with a reduced size and then kill it during the impact with something.

I don't know if the anticheat would trigger, so I don't know if it will be possible to do.

File: CustomItems/Items/Scp2818.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        public override string Name { get; set; } = "SCP-2818";
+
+        /// <inheritdoc/>
+        public override string Description { get; set; } =
+            "When this weapon is fired, it uses the biomass of the shooter as the bullet.";
+
+        /// <inheritdoc/>
+        public override SpawnProperties SpawnProperties { get; set; } = new SpawnProperties
+        {
+            Limit = 1,
```
Comment (iopietro) [inline CustomItems/Items/Scp2818.cs:29]: Shouldn't this be set at 2?

Comment (galaxy119) [reply CustomItems/Items/Scp2818.cs:29]: Why?

**Comment by galaxy119** [comment]
We could have it shoot the player with a reduced size, so long as we are using Player.Position setter which uses PlayerMovementSync.OverridePosition it won't trigger anticheat, similar to how Implosion grenade doesn't when you are teleporting people into the air if it explodes above them.

File: CustomItems/Items/Scp2818.cs (hunk 1 above)
Comment (iopietro) [reply CustomItems/Items/Scp2818.cs:29]: Because the item inside the HCZ harmory would never have a chance to spawn.

**Comment by iopietro** [comment]
> 
> 
> We could have it shoot the player with a reduced size, so long as we are using Player.Position setter which uses PlayerMovementSync.OverridePosition it won't trigger anticheat, similar to how Implosion grenade doesn't when you are teleporting people into the air if it explodes above them.

It would be a cool thing to do, instead of making players to kill themselves and nothing else.

File: CustomItems/Items/Scp2818.cs (hunk 1 above)
Comment (galaxy119) [reply CustomItems/Items/Scp2818.cs:55]: Corrected the spawn chances so that both may have a chance to spawn, but neither are guaranteed.

**Pull request merged**